#! /bin/sh /usr/share/dpatch/dpatch-run
## 19_cups_workaround.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change PS magic string from PS-Adobe-2.0 to PS to work around
## DP: incompatibilities with CUPS (Closes: #443832).

@DPATCH@
diff -urNad transfig~/fig2dev/dev/genps.c transfig/fig2dev/dev/genps.c
--- transfig~/fig2dev/dev/genps.c
+++ transfig/fig2dev/dev/genps.c
@@ -619,7 +619,7 @@
 	if (epsflag)
 	    fprintf(tfp, "%%!PS-Adobe-2.0 EPSF-2.0\n");	/* Encapsulated PostScript */
 	else
-	    fprintf(tfp, "%%!PS-Adobe-2.0\n");		/* PostScript magic strings */
+	    fprintf(tfp, "%%!PS\n");		/* PostScript magic strings */
 
 #ifdef DISPLAYWHO
 	who = getpwuid(getuid());
